### PR TITLE
Add All Portfolios Page and Single Portfolio page

### PR DESCRIPTION
**Pull request by lgalis**
All Portfolios Page:

screenshot from 2018-11-20 10-15-10

Portfolio with actions toolbar:
screenshot from 2018-11-20 15-32-45

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,6 @@
     "react-modal": "^3.5.1",
     "react-spinners": "^0.3.2",
     "react-transition-group": "^2.4.0",
-    "redux-logger": "^3.0.6",
```
Comment (hstastna) [inline package.json]: What is the reason to remove redux logger?

File: src/PresentationalComponents/Portfolio/portfoliocard.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
+@import "../../App.scss";
+
+.pcard_style {
+  margin-top: 0px;
```
Comment (hstastna) [inline src/PresentationalComponents/Portfolio/portfoliocard.scss]: What is the reason to put every option to the single line and define the values like this?
Maybe

margin: 0px;
padding: 0px;

could work, too. https://www.w3schools.com/Css/css_margin.asp

File: package.json (hunk 1 above)
Comment (lgalis) [reply package.json]: it was a merge issue, it's back in

File: src/PresentationalComponents/Portfolio/PorfolioCard.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            <GridItem GridItem sm={ 6 } md={ 4 } lg={ 4 } xl={ 3 }>
+                <Card className="pcard_style">
+                    <CardHeader className="pcard_header">
+                        <ImageWithDefault src={ this.props.imageUrl || DefaultPortfolioImg } defaultSrc={ DefaultPortfolioImg } />
+                    </CardHeader>
+                    <CardBody className="pcard_body">
+                        <h4>{ this.props.name }</h4>
+                        { itemDetails(this.props, TO_DISPLAY) }
+                    </CardBody>
+                    <CardFooter/>
```
Comment (hstastna) [inline src/PresentationalComponents/Portfolio/PorfolioCard.js]: Do we need  `<CardFooter/>` here?

File: src/Store/ActionTypes.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ export const FILTER_PLATFORM_ITEMS = 'FILTER_PLATFORM_ITEMS';
 export const FETCH_PORTFOLIOS = 'FETCH_PORTFOLIOS';
 export const FETCH_PORTFOLIO_ITEMS = 'FETCH_PORTFOLIO_ITEMS';
 export const FETCH_PORTFOLIO_ITEMS_WITH_PORTFOLIO = 'FETCH_PORTFOLIO_ITEMS_WITH_PORTFOLIO';
-export const FETCH_PORTFOLIO_ITEM = 'FETCH_PORTFOLIO_ITEM';
+export const FETCH_PORTFOLIO = 'FETCH_PORTFOLIO';
```
Comment (hstastna) [inline src/Store/ActionTypes.js]: It's better to divide action types into more files, not to have them in only one file like this. The same as we have actions in `AlertActions.js`,`OrderActions.js`, `PlatformActions.js`,... We don't have them all in something like `Actions.js` (and the same for reducers) :)

File: src/PresentationalComponents/Portfolio/PorfolioCard.js (hunk 3 above)
Comment (lgalis) [reply src/PresentationalComponents/Portfolio/PorfolioCard.js]: Not at the moment, but we'll need to add details, so I will leave it in.

File: src/Store/ActionTypes.js (hunk 4 above)
Comment (lgalis) [reply src/Store/ActionTypes.js]: We only have a few at the moment,. Agree, as we have more actions, we should divide the file into store categories.

**Comment by lgalis** [comment]
@hstatsna - I didnot notice the latest UX has teh actions switched - thanks - I will change that in the next PR with the rest of the toolbars.

**Comment by lgalis** [comment]
Most of the codeclimate warnings are caused by the React's propTypes declarations being marked as duplicate. Codeclimate has issue #89 opened for this, with some changes made to allow to filter these out. We could look into configuring a filter for these.

File: src/Store/Actions/PortfolioActions.js [hunk 5]
```diff
@@ -40,7 +40,12 @@ export const searchPortfolioItems = value => ({
...
 
-export const addPortfolioWithItem = (portfolioData, itemData) => ({
+export const addPortfolio = (portfolioData, items) => ({
     type: ActionTypes.ADD_PORTFOLIO,
-    payload: PortfolioHelper.addPortfolioWithItem(portfolioData, itemData)
+    payload: PortfolioHelper.addPortfolio(portfolioData, items)
+});
+
+export const updatePortfolio = (portfolioData) => ({
+    type: ActionTypes.UPDATE_PORTFOLIO,
```
Comment (hstastna) [inline src/Store/Actions/PortfolioActions.js]: You are missing UPDATE_PORTFOLIO in ActionTypes.

File: src/Store/ActionTypes.js (hunk 4 above)
Comment (hstastna) [inline src/Store/ActionTypes.js]: If you do such changes, it is also good to remove references also in _src/Store/Reducers/PortfolioStore.js_.

File: src/Store/Actions/PortfolioActions.js (hunk 5 above)
Comment (lgalis) [reply src/Store/Actions/PortfolioActions.js]: The API for this does not exist yet - I will add the Action type when the API is actually available.

Comment (hstastna) [reply src/Store/Actions/PortfolioActions.js]: No worries, I already added the type in my actual open PR => an ugly warning disappeared and everything works as before.

**Pull request merged**